app/src/components/components/BulkActions/BulkActions.tsx

```diff
@@ -103,13 +103,21 @@ export const BulkActions = () => {
     setData(newData);
   };
 
-  const handleChange = (_, i: number, checked: boolean) => {
+  const handleChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    i: number,
+    checked: boolean
+  ) => {
     const newData = [...data];
     newData[i + pageSize * page].checked = checked;
     setData(newData);
   };
 
-  const handleAction = (_, __, action: HvActionGeneric | HvListValue) => {
+  const handleAction = (
+    event: React.SyntheticEvent,
+    id: string,
+    action: HvActionGeneric | HvListValue
+  ) => {
```

Comment:
We shouldn't be "redefining" the types. It's extra work, prone to errors, less future-proof, etc.

```suggestion
  const handleAction: HvBulkActions["actionsCallback"] = (event, id, action) => {
```

Reply:
You're right, I will change this 🙏